data/Pandora.Data/Transformers/TerraformResourceDefinitionTests.cs

```diff
@@ -36,6 +36,9 @@ public void MappingABasicResourceDefinition()
         Assert.AreEqual("SomeRead", actual.ReadMethod.MethodName);
         Assert.AreEqual(11, actual.ReadMethod.TimeoutInMinutes);
         Assert.AreEqual("fake_planet", actual.ResourceLabel);
+        Assert.AreEqual("fake category", actual.ResourceCategory);
```

Comment:
Test failure typo here:
```suggestion
        Assert.AreEqual("fake resource category", actual.ResourceCategory);
```

Reply:
🤦 thanks for that